public/manifest.json

```diff
@@ -17,5 +17,6 @@
 	],
 	"theme_color": "#7cc0ff",
 	"background_color": "#7cc0ff",
-	"display": "standalone"
+	"display": "standalone",
+	"gcm_sender_id": "103953800507"
```

Comment:
`gcm_sender_id` is the same for ALL Firebase Messaging projects. I'm inclined to leave this as a constant since it won't change for different deploys. Check info here:
https://firebase.google.com/docs/cloud-messaging/js/client